CONTRIBUTING.md

```diff
@@ -254,107 +587,128 @@ The first line is the subject and should be no longer than 70 characters, the
+  timeframe, should a non-core contributor request to be assigned to a `critical` issue,
+  they will be paired with a core team-member to manage the tracking, communication and release of any fix
+  as well as to assume responsibility of all progess._
 
-    b) If this is a subsequent release candidate or the release after an RC check out the existing release branch
+- [`important-soon`](https://github.com/weaveworks/eksctl/labels/priority%2Fimportant-soon): Must be assigned as soon as capacity becomes available.
+  Ideally something should be delivered in time for the next release.
 
-     ```console
-     git checkout release-0.13
-     ```
+- [`important-longterm`](https://github.com/weaveworks/eksctl/labels/priority%2Fimportant-longterm): Important over the long term, but may not be currently
+  staffed and/or may require multiple releases to complete.
 
-1. Create the tags so that circleci can start the release process:
+- [`backlog`](https://github.com/weaveworks/eksctl/labels/priority%2Fbacklog): There appears to be general agreement that this would be good to have,
+  but we may not have anyone available to work on it right now or in the immediate future.
+  PRs are still very welcome, although it might take a while to get them reviewed if
+  reviewers are fully occupied with higher priority issues, for example immediately before a release.
 
-   - for a release candidate: `make prepare-release-candidate`. If there was an existing RC it will increase it's number, e.g.: rc.0 -> rc.1
+- [`needs-investigation`](https://github.com/weaveworks/eksctl/labels/needs-investigation):  There is currently insufficient information to either categorize properly,
+  or to understand and implement a solution. This could be because the issue opener did
+  not provide enough relevant information, or because more in-depth research is required
+  before work can begin.
 
-   - for a release: `make prepare-release`. Regardless of whether there was a previous RC or a development version it will create a normal release
+These priority categories have been inspired by [the Kubernetes contributing guide](https://github.com/kubernetes/community/blob/master/contributors/guide/issue-triage.md).
 
-1. Ensure release jobs succeeded in [CircleCI](https://circleci.com/gh/weaveworks/eksctl).
-1. Ensure the release was successfully [published in Github](https://github.com/weaveworks/eksctl/releases).
-1. Download the binary just released, verify its checksum, and perform any relevant manual testing.
+## Reviewing PRs
 
-### Releasing snaps of eksctl
+The core team aims to clear the PR queue as quickly as possible. Community members
+should also feel free to keep an eye on things and provide their own thoughts and expertise.
 
-Snaps are software packages which run on a variety of Linux flavours.
+---
 
-*Note:* This is still somewhat [TBD](https://github.com/weaveworks/eksctl/issues/215).
+# Proposals
```

Comment:
can we get a markdown template for proposals?

Reply:
sure 👍 . I can submit that in a separate PR to give people more space to consider the format

Reply:
sounds good 👍